tests/context/test_case_context.py

```diff
@@ -0,0 +1,189 @@
+from response_operations_ui.contexts.case import build_response_status_context
+
+
+def test_response_status_context_with_permissions(
```

Comment:
These are fine, and one for another time, but pytest.mark.parametrize would of worked well here and saved a lot of repetition, https://docs.pytest.org/en/7.1.x/how-to/parametrize.html#parametrize. You would of had to use something like getfixturevalue in the function for the compare